Formula/o/openjdk@8.rb

```diff
@@ -123,25 +141,37 @@ def install
         --with-toolchain-type=clang
         --with-zlib=system
       ]
+      # Work around Xcode 16 bug: https://bugs.openjdk.org/browse/JDK-8340341
+      DevelopmentTools.clang_build_version == 1600 &&
+        ENV.append_to_cflags("-mllvm -enable-constraint-elimination=0")
```

Comment:
Refer to issue #195456 for the fix.

P.S. [JDK-8340341](https://bugs.openjdk.org/browse/JDK-8340341)

Reply:
The error log output is similar to the following:
```
## Finished images (build time 00:00:33)

Boot cycle build step 2: Building a new JDK image using previously built image
Warning: No SCM configuration present and no .src-rev
Building OpenJDK for target 'images' in configuration 'release'

## Starting langtools
Compiling 2 files for BUILD_TOOLS
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x0000000109d6c3f5, pid=53877, tid=0x0000000000003803
#
# JRE version: OpenJDK Runtime Environment (8.0_432) (build 1.8.0_432-Homebrew_2024_10_29_14_04-b00)
# Java VM: OpenJDK 64-Bit Server VM (25.432-b00 mixed mode bsd-amd64 compressed oops)
# Problematic frame:
# V  [libjvm.dylib+0x913f5]  ChunkPool::allocate(unsigned long, AllocFailStrategy::AllocFailEnum)+0x2f
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /private/tmp/openjdkA8-20241029-5576-j2dt8y/jdk8u-jdk8u432-ga/langtools/make/hs_err_pid53877.log
#
# If you would like to submit a bug report, please visit:
#   https://github.com/Homebrew/homebrew-core/issues
#
/bin/sh: line 1: 53877 Abort trap: 6           /private/tmp/openjdkA8-20241029-5576-j2dt8y/jdk8u-jdk8u432-ga/build/release/images/j2sdk-image/bin/javac -XDignore.symbol.file=true -g -Xlint:all,-deprecation -Werror -Xprefer:source -implicit:none -sourcepath "/private/tmp/openjdkA8-20241029-5576-j2dt8y/jdk8u-jdk8u432-ga/langtools/make/tools:/private/tmp/openjdkA8-20241029-5576-j2dt8y/jdk8u-jdk8u432-ga/langtools/src/share/classes" -d /private/tmp/openjdkA8-20241029-5576-j2dt8y/jdk8u-jdk8u432-ga/build/release/bootcycle-build/langtools/btclasses @/private/tmp/openjdkA8-20241029-5576-j2dt8y/jdk8u-jdk8u432-ga/build/release/bootcycle-build/langtools/btclasses/_the.BUILD_TOOLS_batch.tmp
make[2]: *** No rule to make target `all', needed by `default'.  Stop.
make[1]: *** [langtools-only] Error 2
make: *** [bootcycle-images-only] Error 2
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/git --version
/usr/bin/env /usr/local/Homebrew/Library/Homebrew/shims/shared/curl --version
/usr/local/Homebrew/Library/Homebrew/ignorable.rb:27:in `block in raise'
BuildError: Failed executing: make bootcycle-images CONF=release
```